lib/util/template/index.ts

```diff
@@ -6,6 +6,10 @@ import { clone } from '../clone';
 
 handlebars.registerHelper('encodeURIComponent', encodeURIComponent);
 
+handlebars.registerHelper('stringToPrettyJSON', (input: string): string =>
```

Comment:
Please split this out to a separate pr.

Reply:
I would like to keep this in here, because I would like to use it in my "personal" prs to include a pretty representation of the lookup name in my pull requests. As you can see, the lookup name is a string representation of JSON... so I think this is related to this PR.
I implemented this earlier, but than removed it because @rarkins wanted to keep the default behavior for the DataSource. But I would like to use it in my personal settings - but think there it would not be possible extend this functionality via config.

Reply:
I like to split this out to a separate PR to be merged before this.

So simply open a new PR with this change only. Then we merge that and afterwards you can merge main again and this will be resolved. 🤗

Reply:
Ok, here we go: https://github.com/renovatebot/renovate/pull/12643